helpers/changed_managed_files.sh

```diff
@@ -26,12 +28,18 @@
 #   S.5......  c /etc/ntp.conf
 
 if [ $UID -ne "0" ]; then
-   SUDOPREFIX="sudo"
+   SUDOPREFIX="sudo -n"
 fi
 
 for package in `rpm -qa --queryformat "%{NAME}-%{VERSION}\\n"`; do
-  CHANGES=`$SUDOPREFIX rpm -V --nodeps --nodigest --nosignature --nomtime --nolinkto $package`;
-  if [ -n "$CHANGES" ]; then
-    echo -e "$package:\\n$CHANGES";
+  # rpm returns 1 as exit code when modified config files are detected
+  # that's why we explicitly detect if sudo failed
+  OUTPUT=`$SUDOPREFIX rpm -V --nodeps --nodigest --nosignature --nomtime --nolinkto $package 2>&1 || true`;
+  if [ -n "$OUTPUT" ]; then
+    if [[ "$OUTPUT" ==  *"password is required"* ]]; then
```

Comment:
This is way to dangerous that somewhere in the regular line an output could appear like this.

I would check for the exact output if it is the same on each distribution and of course make sure that the correct language environment is set.

"sudo: a password is required"